registration/views.py

```diff
@@ -17,49 +17,60 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 """
-Beta test views
+Registration views
 """
 
 # Imports #####################################################################
 
+from django.contrib.auth import authenticate, login
 from django.core.urlresolvers import reverse_lazy
 from django.db import transaction
-from django.shortcuts import render
-from django.views.generic.edit import CreateView
+from django.utils.decorators import method_decorator
+from django.views.decorators.debug import sensitive_post_parameters
+from django.views.generic.edit import UpdateView
+from simple_email_confirmation.models import EmailAddress
 
-from betatest.forms import BetaTestApplicationForm
 from email_verification import send_email_verification
-from simple_email_confirmation.models import EmailAddress
+from registration.forms import BetaTestApplicationForm
 
 
 # Views #######################################################################
 
-class BetaTestApplicationView(CreateView):
+@method_decorator(sensitive_post_parameters('password',
+                                            'password_confirmation'),
+                  name='post')
```

Comment:
@omarkhan nit: maybe move `sensitive_post_parameters` to next line:

    @method_decorator(
        sensitive_post_parameters('password', 'password_confirmation'), name='post'
    )